### DISCUSSION TITLE
List of messages for each device

### DISCUSSION THREAD

**Discussion by dac2020** [arendst/Tasmota#16606] [Q&A] [answered]
Is there a way to generate all of the possible messages that each device I'm using might publish?

**Comment by sfromis** [comment, marked as answer]
No. Closest is to get it into the situations where publishing occurs. However, there is generally not much in surprises, as predictability is high when you know what you're doing with the device.

**Reply by dac2020** [reply]
Thank you. Using status commands for example seems to give most everything needed.

**Reply by sfromis** [reply]
Not at all.
